frontend/ui/widget/infomessage.lua

```diff
@@ -93,8 +93,9 @@ function InfoMessage:init()
             }
         else
             image_widget = ImageWidget:new{
-                file = "resources/info-i.png",
+                file = self.file or "resources/info-i.png",
```

Comment:
Mind calling it `icon_file` and documenting it in the constructor?
```diff
     image_height = nil,  -- The image height if image is used. Keep it nil to use original height.
     -- Whether the icon should be shown. If it is false, self.image will be ignored.
     show_icon = true,
+    icon_file = nil, -- use this file instead of "resources/info-i.png"
     dismiss_callback = function() end,
```
(not `image_file` , we'll use that in case we want to use an image file that will obey `image_height` and `image_width`, without `scale_for_dpi=true`)